packages/amplify-e2e-tests/src/__tests__/api_6a.test.ts

```diff
@@ -9,36 +9,61 @@ import {
   scanTable,
   rebuildApi,
   getProjectMeta,
+  updateApiSchema,
 } from '@aws-amplify/amplify-e2e-core';
 
 const projName = 'apitest';
+
+export const testTableBeforeRebuildApi = async (apiId: string, region: string, modelName: string) => {
```

Comment:
nit: I see these methods `testTableBeforeRebuildApi` and `testTableAfterRebuildApi` could be used in both e2e tests, can you move these to a shared test utils?